packages/indexer-agent/src/agent.ts

```diff
@@ -371,7 +401,65 @@ export class Agent {
       {
         onError: error =>
           this.logger.warn(
-            `Failed to obtain target allocations, trying again later`,
+            `Failed to evaluate deployments, trying again later`,
+            {
+              error,
+            },
+          ),
+      },
+    )
+
+    // Update targetDeployments and networkDeplomentAllocationDecisions using transferredSubgraphDeployments data
+    // This will be somewhat custom and will likely be yanked out later after the transfer stage is complete
+    // Cases:
+    // L1 subgraph that has been transferred: keep synced and allocated to for at least 24 hours post transfer
+    // L2 subgraph that has been transferred:
+    //     if already synced, allocate to it immediately using default allocation amount
+    //     if not synced, no changes
+    const networkDeploymentAllocationDecisions: Eventual<NetworkMapped<AllocationDecision[]>> = join({
+      intermediateNetworkDeploymentAllocationDecisions,
+      eligibleTransferDeployments,
+    }).tryMap(
+        ({ intermediateNetworkDeploymentAllocationDecisions, eligibleTransferDeployments }) => {
+        const zipped = this.multiNetworks.zip(
+          intermediateNetworkDeploymentAllocationDecisions,
+          eligibleTransferDeployments,
+        )
+
+        mapValues(
+          zipped,
+          async ([allocationDecisions, eligibleTransferDeployments]: [
+            AllocationDecision[],
+            TransferredSubgraphDeployment[],
+          ]) => {
+
+            allocationDecisions.map(async decision => {
+              const matchingTransfer = eligibleTransferDeployments.find(deployment => deployment.ipfsHash == decision.deployment.ipfsHash)
+
+              if (matchingTransfer) {
+                const dayAgo = Math.floor(Date.now() / 1000) - 86400
+                const fourDaysAgo = Math.floor(Date.now() / 1000) - 86400 * 4
+                // L1 deployments being transferred need to be supported for 24hrs post transfer to ensure continued support
+                if (+matchingTransfer.protocolNetwork < 10 && matchingTransfer.transferredToL2At.toNumber() > dayAgo) {
+                    decision.toAllocate = true
+                }
+                // L2 Deployments
+                if (+matchingTransfer.protocolNetwork > 10 && matchingTransfer.transferredToL2At.toNumber() > fourDaysAgo) {
+                    decision.toAllocate = true
+                }
+              }
+            })
+
+            return allocationDecisions
+          },
+        )
+
+        return a
```

Comment:
I think you meant this, right?
```suggestion
        return allocationDecisions```